opslevel/datasource_opslevel_webhook_action.go

```diff
@@ -24,17 +26,29 @@ type WebhookActionDataSource struct {
 	CommonDataSourceClient
 }
 
-// WebhookActionDataSourceModel describes the data source data model.
-type WebhookActionDataSourceModel struct {
+type webhookActionWithIdentifierDataSourceModel struct {
```

Comment:
Needed one with and without `Identifier`

Also added fields that exist: `Id` and `Aliases`.